tools/Github/ParseWiki2Json.ps1

```diff
@@ -0,0 +1,79 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+    Sync ADO wiki aliases content to fabricbot.json.
+
+#>
+param(
+    [Parameter(Mandatory = $true)]
+    [string]$AzureToken
+)
+
+# get wiki content
+$username=""
+$password=$AzureToken
+$pair="{0}:{1}" -f ($username,$password)
+$bytes = [System.Text.Encoding]::ASCII.GetBytes($pair)
+$token = [System.Convert]::ToBase64String($bytes)
+$headers = @{
+    Authorization = "Basic {0}" -f ($token)
+}
+
+$response = Invoke-RestMethod 'https://dev.azure.com/azure-sdk/internal/_apis/wiki/wikis/internal.wiki/pages?api-version=7.1-preview.1&path=/Engineering%20System/GitHub%20Repos/Issue%20Management/Service%20Team%20Label%20and%20Contact%20List&includeContent=True' -Headers $headers
+
+$rows = ($response.content -split "\n") | Select-Object -Skip 6
```

Comment:
Not a good idea to rely on the magic number. You can try use `ConvertFrom-Markdown` to parse this document.

Reply:
I have changed it to be: `$rows = ($response.content -split "\n") | Where-Object { $_ -like '|*'} | Select-Object -Skip 2`, which is more robust for potential changes. The number 2 here is to skip the table header and separator.